apps/webpack.js

```diff
@@ -18,6 +18,9 @@ var baseConfig = {
       '@cdo/weblab/locale': path.resolve(__dirname, 'src', 'weblab', 'locale-do-not-import.js'),
       '@cdo/apps': path.resolve(__dirname, 'src'),
       repl: path.resolve(__dirname, 'src/noop'),
+      'jquery': path.resolve(__dirname, 'node_modules', 'jquery', 'dist', 'jquery.js'),
+      '@cdo/dashboard': path.resolve(__dirname, '..', 'dashboard'),
+      '@cdo/shared': path.resolve(__dirname, '..', 'shared'),
```

Comment:
the above two lines make me cry a bit, since we are now depending on things in a sibling directory!